src/main/java/gregtech/api/unification/crafttweaker/CTMaterialBuilder.java

```diff
@@ -85,6 +85,19 @@ public CTMaterialBuilder gem(@Optional int harvestLevel, @Optional int burnTime)
         return this;
     }
 
+    @ZenMethod
+    public CTMaterialBuilder polymer() {
+        backingBuilder.polymer();
+        return this;
+    }
```

Comment:
Unnecessary with the below method that is optional (and I think may even cause issues for CT)